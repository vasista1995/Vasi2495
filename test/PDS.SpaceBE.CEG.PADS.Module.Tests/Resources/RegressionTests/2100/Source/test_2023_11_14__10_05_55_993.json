{
  "IdSource": "SPACEACT3:CEG:BE:81755470:Econo:system soldering:K2-t-HCOOH:101346:332712253:0",
  "Data_flat_Metadata": {
    "Plant": "0600",
    "Line": "Econo",
    "ProductionOrder": "81755470",
    "Batch": "H27421P0",
    "MaterialNumber": "46964",
    "ProductGroup": "M-ECONOSTECK",
    "MeasurementRecipe": "5",
    "ProcessEquipment": "132013101041",
    "OrderType": "2100",
    "QrkGroup": "2100",
    "ProcessGroup": "system_soldering",
    "SpcClass": "2100",
    "ParameterType": "2100",
    "LongParameterName": "K2-Flutzeit HCOOH",
    "MaterialNumberText": "2100",
    "ProcessEquipmentName": "VADU25",
    "Operator": "2100",
    "ParameterClass": "NON_PPAP",
    "Segment": "HPS",
    "Package": "ECONO",
    "Module": "11 substrate-baseplate interconnect",
    "PosName": "2100",
    "FourDReport": "*",
    "ParameterName": "K2-t-HCOOH",
    "ParameterUnit": "s",
    "ParameterOper": "system soldering",
    "SiteKey": "CEG",
    "ChannelId": "101346",
    "SpaceInstanceName": "BE",
    "CreatedTimestampUtc": "2023-10-27T09:19:53Z",
    "CreatedTimestamp": "2023-10-27T11:19:53Z",
    "UpdatedTimestamp": "2023-10-27T11:19:53Z",
    "UpdatedTimestampUtc": "2023-10-27T09:19:53Z",
    "ExportedTimestampUtc": "2023-10-27T11:14:48Z",
    "ExportedTimestamp": "2023-10-27T11:14:48Z",
    "SampleTimestampUtc": "2023-10-27T09:14:48Z",
    "SampleTimestamp": "2023-10-27T11:14:48Z",
    "SourceDataLevel": "C",
    "ChannelName": "K2-t-HCOOH /Activation_target_55/VADU25",
    "RvStoreFlag": "Y",
    "CalcCreated": "01/25/2023 05:38:12",
    "ILdsID": 0,
    "AcceptFlag": "Y",
    "UpdatedBy": " ",
    "ClOrigin": 0,
    "OutOfOrder": "N",
    "Calc2ID": 9685588,
    "ViolCount": 0,
    "CalculationStrategy": "ND 4.5 Sigma",
    "LdsID": "2100",
    "IsCalcParam": 0,
    "ChannelType": "N",
    "ChState": "Online"
  },
  "Data_1list_RawValues": {
    "Data_flat_Limits": {
      "CkcId": "0",
      "MeasurementSpecLimits": {
        "SpecHighEnabled": "Y",
        "SpecLowEnabled": "Y",
        "SpecHigh": 68.0,
        "SpecLow": 42.0,
        "SpecTarget": 55.0,
        "SpecTargetOrigin": "V",
        "ExtSpecLimEnable": "NN",
        "ExtSpecUpper": 0.0,
        "ExtSpecLower": 0.0,
        "LimitEnable": "YYNNNNYNNYYYYYYYYYYNNNYYYYYYNNNNNNNNN"
      },
      "SpaceAggregates": {
        "Mean": "55,3",
        "Median": "55,3",
        "Sigma": "0",
        "Min": "55,3",
        "Max": "55,3",
        "Q1": "55,3",
        "Q3": "55,3"
      },
      "ControlLimits": {
        "ctrl_mean_low": 53.6663685165165,
        "CtrlMeanLowEnabled": "Y",
        "ctrl_mean_target": 55.0,
        "CtrlMeanTargetEnabled": "Y",
        "ctrl_mean_high": 56.3336314834835,
        "CtrlMeanHighEnabled": "Y",
        "CtrlLowEnabled": "N",
        "CtrlSigmaCenterEnabled": "N",
        "CtrlRangeCenterEnabled": "N",
        "CtrlCenterEnabled": "N",
        "CtrlHighEnabled": "N",
        "CtrlSigmaLowEnabled": "N",
        "CtrlSigmaHighEnabled": "N",
        "CtrlRangeLowEnabled": "N",
        "CtrlRangeHighEnabled": "N",
        "ExtMaMV": 55.0714285714286,
        "ExtEwmaMV": 55.2058186643449,
        "ExtMsMV": 0.0707106781186507,
        "EwmaS": 0.0,
        "EwmaR": 0.0,
        "ExtEwmaMVLCL": 54.5562355481985,
        "ExtEwmaMVCenter": 55.0008485639686,
        "ExtEwmaMVUCL": 55.4454615797387,
        "ExtMaMVLCL": 54.6647527155714,
        "ExtMaMVCenter": 55.0008485639686,
        "ExtMaMVUCL": 55.3369444123658,
        "EwmaSLCL": 0.0,
        "EwmaSCenter": 0.0,
        "EwmaSUCL": 0.0,
        "EwmaRLCL": 0.0,
        "EwmaRCenter": 0.0,
        "EwmaRUCL": 0.0,
        "ExtMSMVLCL": 0.00046580646315045,
        "ExtMSMVCenter": 0.219660342999546,
        "ExtMSMVUCL": 0.882384104550397,
        "MVLal": 0.0,
        "MVUal": 0.0,
        "RawLal": 53.6663685165165,
        "RawUal": 56.3336314834835,
        "SigmaLal": 0.0,
        "SigmaUal": 0.0,
        "RangeLal": 0.0,
        "RangeUal": 0.0
      }
    },
    "Data_1list_RawValues": [
      {
        "SampleId": 332712253,
        "Seqnr": 1,
        "Value": 55.3,
        "InternalFlagged": "N",
        "ExternalFlagged": "N",
        "NumViolations": 0
      }
    ],
    "SampleSize": 1
  },
  "ProductionAction": {
    "Id": "SPACEACT1:CEG:BE:Econo::K2-t-HCOOH:101346",
    "Type": "SpaceAction"
  },
  "Item": {
    "Id": "HPSModFAUF:81755470",
    "IdSystemName": "HPSModFAUF",
    "Type": "Fauf"
  },
  "SystemLog": {
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "UniqueSourceKey": "CEG:BE:81755470:Econo:system soldering:K2-t-HCOOH:101346:332712253:0"
  }
}