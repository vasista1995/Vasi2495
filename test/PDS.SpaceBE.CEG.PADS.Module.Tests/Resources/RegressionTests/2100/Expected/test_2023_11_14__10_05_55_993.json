{
  "Data_flat_Metadata": {
    "Line": "Econo",
    "ProductionOrder": "81755470",
    "Batch": "H27421P0",
    "MaterialNumber": "46964",
    "ProductGroup": "M-ECONOSTECK",
    "MeasurementRecipe": "5",
    "OrderType": "2100",
    "QrkGroup": "2100",
    "ProcessGroup": "system_soldering",
    "SpcClass": "2100",
    "CustomComment": "system soldering",
    "Segment": "HPS",
    "MeasLot": "81755470",
    "ParameterOper": "system soldering",
    "SiteKey": "CEG",
    "SpaceInstanceName": "BE",
    "BeginTimestampUtc": "2023-10-27T09:14:48Z",
    "BeginTimestamp": "2023-10-27T11:14:48Z",
    "EndTimestampUtc": "2023-10-27T09:14:48Z",
    "EndTimestamp": "2023-10-27T11:14:48Z"
  },
  "Data_flat_StrangeMetadata": {
    "MaterialNumberText": "2100",
    "Operator": "2100",
    "ParameterClass": "NON_PPAP",
    "Package": "ECONO",
    "Module": "11 substrate-baseplate interconnect",
    "PosName": "2100"
  },
  "Data_1list_Parameters": [
    {
      "process_equipment": "132013101041",
      "ProcessEquipmentName": "VADU25",
      "LongParameterName": "K2-Flutzeit HCOOH",
      "Data_1list_RawValues": [
        {
          "process_equipment": "132013101041",
          "ProcessEquipmentName": "VADU25",
          "Seqnr": 1,
          "SampleId": 332712253,
          "Value": 55.3,
          "is_flagged": "N",
          "SampleTimestamp": "2023-10-27T11:14:48Z",
          "SampleTimestampUtc": "2023-10-27T09:14:48Z",
          "CreatedTimestamp": "2023-10-27T11:19:53Z",
          "CreatedTimestampUtc": "2023-10-27T09:19:53Z",
          "UpdatedTimestamp": "2023-10-27T11:19:53Z",
          "UpdatedTimestampUtc": "2023-10-27T09:19:53Z",
          "NumViolations": 0,
          "SampleMean": 55.3,
          "SampleMin": 55.3,
          "SampleMax": 55.3,
          "SampleMedian": 55.3,
          "SampleStdev": 0.0,
          "SampleQ1": 55.3,
          "SampleQ3": 55.3,
          "SampleSize": 1
        }
      ],
      "ParameterType": "2100",
      "FourDReport": "*",
      "Data_flat_Limits": {
        "CKC_ID": "0",
        "MeasurementSpecLimits": {
          "SpecHighEnabled": "Y",
          "SpecLowEnabled": "Y",
          "spec_high": 68.0,
          "spec_low": 42.0,
          "target": 55.0
        },
        "ControlLimits": {
          "ctrl_mean_low": 53.6663685165165,
          "CtrlMeanLowEnabled": "Y",
          "ctrl_mean_target": 55.0,
          "CtrlMeanTargetEnabled": "Y",
          "ctrl_mean_high": 56.3336314834835,
          "CtrlMeanHighEnabled": "Y",
          "CtrlLowEnabled": "N",
          "CtrlHighEnabled": "N",
          "CtrlStdevLowEnabled": "N",
          "CtrlSigmaCenterEnabled": "N",
          "CtrlRangeCenterEnabled": "N",
          "CtrlCenterEnabled": "N",
          "CtrlStdevHighEnabled": "N",
          "CtrlRangeLowEnabled": "N",
          "CtrlRangeHighEnabled": "N"
        }
      },
      "Data_flat_MeasurementAggregates": {
        "exec_count": 1,
        "base_count": 1,
        "flagged_count": 0,
        "mean": 55.3,
        "median": 55.3,
        "stdev": 0.0,
        "range": 0.0,
        "min": 55.3,
        "max": 55.3,
        "q02": 55.3,
        "q05": 55.3,
        "q25": 55.3,
        "q75": 55.3,
        "q95": 55.3,
        "q98": 55.3,
        "NumViolations": 0,
        "Samples": "332712253"
      },
      "ParameterName": "K2-t-HCOOH",
      "ParameterUnit": "s",
      "ChannelId": "101346",
      "SourceDataLevel": "C",
      "channel_name": "K2-t-HCOOH  Activation_target_55 VADU25",
      "RvStoreFlag": "Y",
      "created_timestamp_utc": "2023-10-27T09:19:53Z",
      "created_timestamp": "2023-10-27T11:19:53Z",
      "updated_timestamp_utc": "2023-10-27T09:19:53Z",
      "updated_timestamp": "2023-10-27T11:19:53Z"
    }
  ],
  "Document": {
    "Version": "1.0",
    "Type": "ProcessControl",
    "Repetition": {
      "Id": "0",
      "IdBaseValue": "2023-10-27T09:19:53Z"
    }
  },
  "ProductionAction": {
    "Id": "SPACEAGGED3:CEG:BE:Econo:system soldering:",
    "Type": "SpaceAggregation"
  },
  "Item": {
    "Id": "HPSModFAUF:81755470",
    "IdSystemName": "HPSModFAUF",
    "Type": "Fauf"
  },
  "SystemLog": {
    "UniqueSourceKey": "CEG:BE:81755470:Econo:system soldering:K2-t-HCOOH:101346:332712253:0",
    "ExtractedTimestampUtc": "0001-01-01T00:00:00",
    "EnqueuedTimestampUtc": "0001-01-01T00:00:00",
    "DocCreatedTimestampUtc": "2023-10-27T11:14:48Z",
    "QueueOffset": 0,
    "QueuePartition": 0
  },
  "SearchPatterns": {
    "SpaceKey": "HPSModFAUF:81755470:SPACEAGGED3:CEG:BE:Econo:system soldering::ProcessControl:1.0",
    "TimeGroup": "0",
    "SiteKey": "CEG"
  }
}